mlflow/johnsnowlabs.py

```diff
@@ -0,0 +1,786 @@
+"""
+The ``mlflow.johnsnowlabs`` module provides an API for logging and loading Spark NLP and NLU models. This module
+exports the following flavors:
+
+The following Environment Variables must be present
```

Comment:
```suggestion
The following environment variables must be present
```

Reply:
resolved in https://github.com/mlflow/mlflow/pull/8556/commits/76d55dcdd647b6d183e933e3f7802995c95fc2fc